src/org/labkey/test/tests/ExpTest.java

```diff
@@ -24,7 +24,10 @@
 import org.labkey.test.TestTimeoutException;
 import org.labkey.test.categories.DailyB;
 import org.labkey.test.categories.FileBrowser;
+import org.labkey.test.components.DomainDesignerPage;
 import org.labkey.test.components.PropertiesEditor;
```

Comment:
One good indicator that the test has been fully converted is when this PropertiesEditor import can be removed. Is that the case here?

Reply:
removed the import.